torch/_inductor/compile_fx.py

```diff
@@ -332,6 +348,7 @@ def compile_fx_inner(
                 is_inference=is_inference,
             )
         else:
+            log.debug("disabled cudagraphs because %s", cudagraph_fail_reasons)
```

Comment:
Wait wait I think you misunderstood. I just noticed there is preexisting code reporting cudagraphs disablement:

```
            if len(set(graph.device_types)) > 1:
                perf_hint_log.warning("skipping cudagraphs due to multiple devices")
            elif set(graph.device_types) == {"cuda"}:
                if graph.mutated_inputs:
                    perf_hint_log.warning("skipping cudagraphs due to input mutation")
                elif complex_memory_overlap_inputs:
                    perf_hint_log.warning(
                        "skipping cudagraphs due to complex input striding"
                    )
                elif len(graph.device_idxs) > 1 and config.triton.cudagraph_trees:
                    perf_hint_log.warning(
                        "skipping cudagraphs due to multiple device indexes"
                    )
```

This log message here is purely additive; I added it because I ran TORCH_LOGS=+inductor and I didn't see any log telling me if cudagraphs was running or not.  I did not modify the original `perf_hint_logs`.

I'm not really in the mood for bikeshedding, so tell me exactly what you want and I'll do it. If necessary I will just delete this debug line to unblock.